### PR TITLE
rosdistro sync, Wed Mar  7 13:42:10 2018

### PR DESCRIPTION
**Pull request by allenh1**
None
Changes:
========
Ardent Changes:
---------------
* *sros2 .4.0 --> 0.4.1*

Indigo Changes:
---------------
* *ddynamic_reconfigure_python 0.0.1*

Kinetic Changes:
---------------
* *ddynamic_reconfigure_python 0.0.1*
* *libmavconn .23.1 --> 0.23.2*
* *mavlink 2018.2.2 --> 2018.3.7*
* *mavros_extras .23.1 --> 0.23.2*
* *mavros_msgs .23.1 --> 0.23.2*
* *test_mavros .23.1 --> 0.23.2*

Lunar Changes:
---------------
* *libmavconn .23.1 --> 0.23.2*
* *mavlink 2018.2.2 --> 2018.3.7*
* *mavros_extras .23.1 --> 0.23.2*
* *mavros_msgs .23.1 --> 0.23.2*
* *test_mavros .23.1 --> 0.23.2*

Missing Dependencies:
=====================
 * [ ] app1
 * [ ] app2
 * [ ] app3
 * [ ] app_loader
 * [ ] can-utils
 * [ ] disper
 * [ ] geographiclib
 * [ ] geographiclib-tools
 * [ ] gringo
 * [ ] gsdf_msgs
 * [ ] julius-voxforge
 * [ ] language-pack-de
 * [ ] language-pack-en
 * [ ] leg_detector
 * [ ] libcoin80-dev
 * [ ] libesd0-dev
 * [ ] libfreenect-dev
 * [ ] mapnik-utils
 * [ ] micros_swarm
 * [ ] micros_swarm_gazebo
 * [ ] micros_swarm_stage
 * [ ] nodejs-legacy
 * [ ] olfati_saber_flocking
 * [ ] opensplice_dds_broker
 * [ ] pcl
 * [ ] people_velocity_tracker
 * [ ] pso
 * [ ] python-bson
 * [ ] python-catkin-lint
 * [ ] python-catkin-tools
 * [ ] python-chainer-pip
 * [ ] python-chainercv-pip
 * [ ] python-dlib
 * [ ] python-expiringdict
 * [ ] python-fcn-pip
 * [ ] python-freezegun-pip
 * [ ] python-gdown-pip
 * [ ] python-mapnik
 * [ ] python-percol
 * [ ] python-pygithub3
 * [ ] python-slacker-cli
 * [ ] python-speechrecognition-pip
 * [ ] python-tabulate-pip
 * [ ] python-tilestache
 * [ ] python-tk
 * [ ] ros_broker
 * [ ] sr_moveit_config
 * [ ] testbb
 * [ ] testnc
 * [ ] testrth
 * [ ] testscdspso
 * [ ] testvstig
 * [ ] thormang3_action_script_player
 * [ ] thormang3_foot_step_generator
 * [ ] thormang3_offset_tuner_client
 * [ ] xdot

### CODE REVIEW

File: ros-ardent/sros2/sros2-0.4.1.ebuild [hunk 1]
```diff
@@ -8,7 +8,7 @@ inherit ament-python
 
 DESCRIPTION="Command line tools for managing SROS2 keys"
 HOMEPAGE="https://wiki.ros.org"
-SRC_URI="https://github.com/ros2-gbp/${PN}-release/archive/release/ardent/${PN}/0.4.0-0.tar.gz -> ${PN}-ardent-release-${PV}.tar.gz"
+SRC_URI="https://github.com/ros2-gbp/${PN}-release/archive/release/ardent/${PN}/0.4.1-0.tar.gz -> ${PN}-ardent-release-${PV}.tar.gz"
```
Comment (allenh1) [inline ros-ardent/sros2/sros2-0.4.1.ebuild]: @mikaelarguedas :tada: :taco: :tada:

Comment (allenh1) [reply ros-ardent/sros2/sros2-0.4.1.ebuild]: (first ROS 2 update for Gentoo)

Comment (mikaelarguedas) [reply ros-ardent/sros2/sros2-0.4.1.ebuild]: You may want to hold for a day as we're about to sync: https://github.com/ros2/ros2/issues/472

Comment (mikaelarguedas) [reply ros-ardent/sros2/sros2-0.4.1.ebuild]: Nevermind this is already merged ^^

Comment (allenh1) [reply ros-ardent/sros2/sros2-0.4.1.ebuild]: Ya, I'll maybe start following with Debian syncs one day...

**Pull request merged**